http/server.ts

```diff
@@ -229,58 +229,21 @@ function fixLength(req: ServerRequest): void {
 }
 
 // ParseHTTPVersion parses a HTTP version string.
-// "HTTP/1.0" returns (1, 0, true).
-// Ported from https://github.com/golang/go/blob/f5c43b9/src/net/http/request.go#L766-L792
+// "HTTP/1.0" returns (1, 0).
 export function parseHTTPVersion(vers: string): [number, number] {
-  switch (vers) {
-    case "HTTP/1.1":
-      return [1, 1];
+  const httpReg = /^HTTP\/([0-9]{0,2})\.([0-9]{0,2})$/gm;
```

Comment:
This isn't correct. The digit minimum should be 1.

You might know something I don't but also why is it capped at 2? 🤔

Reply:
You'r totally true ! I will use your previous answer !

Reply:
And i was capping at 2 because of the actual version wish cap at 1000000 wish was arbitrary
But 2 was even more arbitrary.